README.md

```diff
@@ -48,12 +48,14 @@ It should _also_ be possible to get the list in JSON format, like this:
 
 In JSON format, the application should list counts for all the words.
 
-We don't want you to spend more than two or three hours on this, so feel free
+We know time is limited, and the goal of this exercise is to show us the
+quality of your work. We'd much rather see a simple version of the problem
```

Comment:
The wording makes me think 'simple version of your solution' might be better here.